### PR TITLE
Add JavaDoc comments, refactor isaacsaw25.md

### PR DESCRIPTION
**Pull request by isaacsaw25**
Add Javadoc comments for the file processing functions in the `UI` class. Refactored isaacsaw25.md to fit the requirements.

### CODE REVIEW

File: src/main/java/ui/UI.java [hunk 1]
```diff
@@ -149,14 +149,17 @@ public void generateReport(ArrayList<Student> studentList) throws SEPEmptyExcept
...
+     */
     public void printConfigMessage() {
-        System.out.println(HORIZONTAL_LINE);
-        System.out.println(Messages.WELCOME);
-        System.out.println("Would you like to:");
-        System.out.println("1. Manually input students data");
-        System.out.println("2. Upload a file (.csv, .txt, .json)");
-        System.out.println("Please choose 1 or 2 or exit :)");
-        System.out.println(HORIZONTAL_LINE);
+        StringBuilder output = new StringBuilder();
```
Comment (ehz0ah) [inline src/main/java/ui/UI.java:156]: Nice

**Review by ehz0ah — APPROVED** [review]
Looks good!

File: src/main/java/ui/UI.java (hunk 1 above)
Comment (isaacsaw25) [reply src/main/java/ui/UI.java:156]: Thanks 😄

Comment (Holy-An) [inline src/main/java/ui/UI.java:156]: Cool

File: src/main/java/ui/UI.java [hunk 2]
```diff
@@ -230,6 +279,9 @@ public boolean toSave() {
         return save.equals("yes");
     }
 
+    /**
+     * Prints a farewell message and shuts down the scanner object.
+     */
```
Comment (Holy-An) [inline src/main/java/ui/UI.java:284]: Wow

**Review by Holy-An — APPROVED** [review]
LGTM

**Review by ehz0ah — APPROVED** [review]
LGTM!

**Review by ehz0ah — APPROVED** [review]
LGTM!  😄 🐱

**Pull request merged**